scopes/component/ui/compare/version-picker/component-compare-version-picker.module.scss

```diff
@@ -3,13 +3,12 @@
 .componentCompareVersionPicker {
   display: flex;
   flex-direction: row;
-  padding: 24px 0px 8px 0px;
+  padding: 8px 0px;
 }
 
-.componentCompareVersionPlaceholder,
 .componentCompareVersionMenu {
-  max-width: 100%;
-  width: 100%;
+  left: 0;
```

Comment:
are you sure this is needed?

Reply:
yes, otherwise the menu is not aligned correctly with the placeholder

![image](https://user-images.githubusercontent.com/8999604/173878789-24bccfec-8ff5-4abc-93e1-8938c3d63dc2.png)